verification/curator-service/ui/src/redux/store.ts

```diff
@@ -2,17 +2,20 @@ import { configureStore, combineReducers } from '@reduxjs/toolkit';
 
 import appReducer from './app/slice';
 import authReducer from './auth/slice';
+import countryListReducer from './filters/slice';
```

Comment:
To make things consistent you can change the name here from `countryListReducer` to `filtersReducer`